command/agent/fs_endpoint_test.go

```diff
@@ -518,3 +521,24 @@ func TestHTTP_FS_Logs_Follow(t *testing.T) {
 		p.Close()
 	})
 }
+
+func TestHTTP_FS_Logs_PropagatesErrors(t *testing.T) {
+	t.Parallel()
+	httpTest(t, nil, func(s *TestAgent) {
+		path := fmt.Sprintf("/v1/client/fs/logs/%s?type=stdout&task=web&offset=0&origin=end&plain=true",
+			uuid.Generate())
+
+		p, _ := io.Pipe()
```

Comment:
Do we need to close the `PipeWriter`?

Reply:
I don't think so; we don't seem to close it in other tests and it's an in memory struct with closed mostly using for signaling errors to other pipe.

Though after looking into it further, the pipe seems unnecessary, so I removed it from this tests and other tests in the file.

Reply:
I don't think so; we don't seem to close it in other tests and it's an in memory struct with closed mostly using for signaling errors to other pipe.

Though after looking into it further, the pipe seems unnecessary, so I removed it from this tests and other tests in the file.